### PR TITLE
[docs] Update changelog (v1.0.1)

### PR DESCRIPTION
**Pull request by AndreyBelym**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
 
+## v1.0.1 (2019-2-15)
+
+### :gear: Package dependencies have been upgraded to avoid CVEs found in the 'lodash' package
+
+### Bug Fixes
+
+* TestCafe no longer hangs when CLI argument validation failed in Live mode ([#3402](https://github.com/DevExpress/testcafe/issues/3402))
+* TestCafe no longer fail with the `ERR_STREAM_WRITE_AFTER_END` error after restarting tests in Live mode ([#3322](https://github.com/DevExpress/testcafe/issues/3322))
+* Now TestCafe doesn't ignore video and encoding specified in a configuration file ([#3415](https://github.com/DevExpress/testcafe/issues/3415))
```
Comment (MargaritaLoseva) [inline CHANGELOG.md]:
```suggestion
* Now TestCafe doesn't ignore video options and encoding specified in a configuration file ([#3415](https://github.com/DevExpress/testcafe/issues/3415))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+* Now you can specify only tests in TestCafe CLI if browsers are specified in a configuration file ([#3421](https://github.com/DevExpress/testcafe/issues/3421))
+* TestCafe no longer stops executing tests in Live mode with multiple browsers when tests don't contain any action ([#3410](https://github.com/DevExpress/testcafe/issues/3410))
```
Comment (MargaritaLoseva) [inline CHANGELOG.md]:
```suggestion
* Live mode: TestCafe no longer stops test execution in multiple browsers if tests do not contain actions ([#3410](https://github.com/DevExpress/testcafe/issues/3410))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+* Now TestCafe correctly handles the `data-parsley-multiple` attribute ([testcafe-hammerhead/#1845](https://github.com/DevExpress/testcafe-hammerhead/issues/1845))
+* Now TestCafe allow passing the `headers` option of the `fetch` function as an Array ([testcafe-hammerhead/#1898](https://github.com/DevExpress/testcafe-hammerhead/issues/1898))
```
Comment (MargaritaLoseva) [inline CHANGELOG.md]:
```suggestion
* Now TestCafe allows passing the `headers` option of the `fetch` function as an Array ([testcafe-hammerhead/#1898](https://github.com/DevExpress/testcafe-hammerhead/issues/1898))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v1.0.1 (2019-2-15)
+
+### :gear: Package dependencies have been upgraded to avoid CVEs found in the 'lodash' package
+
+### Bug Fixes
+
+* TestCafe no longer hangs when CLI argument validation failed in live mode ([#3402](https://github.com/DevExpress/testcafe/issues/3402))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: failed -> fails

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
...
+* TestCafe no longer fails with the `ERR_STREAM_WRITE_AFTER_END` error after restarting tests in live mode ([#3322](https://github.com/DevExpress/testcafe/issues/3322))
+* Now TestCafe doesn't ignore video and encoding options specified in a configuration file ([#3415](https://github.com/DevExpress/testcafe/issues/3415))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: Now TestCafe doesn't ignore -> TestCafe does not ignore

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,22 @@
...
+* Now you can specify only tests in TestCafe CLI if browsers are specified in a configuration file ([#3421](https://github.com/DevExpress/testcafe/issues/3421))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: Now you can -> You can now

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,22 @@
...
+* Live mode: TestCafe no longer stops test execution in multiple browsers if tests do not contain actions ([#3410](https://github.com/DevExpress/testcafe/issues/3410))
+* Now TestCafe correctly handles the `data-parsley-multiple` attribute ([testcafe-hammerhead/#1845](https://github.com/DevExpress/testcafe-hammerhead/issues/1845))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: Now TestCafe correctly -> TestCafe now correctly

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,22 @@
...
+* Now TestCafe allows passing the `headers` option of the `fetch` function as an Array ([testcafe-hammerhead/#1898](https://github.com/DevExpress/testcafe-hammerhead/issues/1898))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: Same

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,22 @@
...
+* No error occurs when page scripts pass a number as an argument to the `window.open` function ([testcafe-hammerhead/#1908](https://github.com/DevExpress/testcafe-hammerhead/issues/1908))
+* Now TestCafe correctly processes rewritten stylesheets ([testcafe-hammerhead/#1919](https://github.com/DevExpress/testcafe-hammerhead/pull/1919))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: Same

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,22 @@
...
+* TestCafe correctly processes sourcemaps in stylesheets now ([testcafe-hammerhead/#1907](https://github.com/DevExpress/testcafe-hammerhead/issues/1907))
```
Comment (dirk-pieterse) [inline CHANGELOG.md]: TestCafe correctly processes sourcemaps in stylesheets now -> TestCafe now correctly processes source maps in stylesheets

**Pull request merged**